### PR TITLE
Set SNMP Trap Handler DIRs via librenms.env

### PR DESCRIPTION
**Pull request by hjcday**
Changes to allow SNMP Trap MIB directories to be set via Env Variables instead of hard coded "/opt/librenms/mibs:/opt/librenms/mibs/cisco"

### CODE REVIEW

File: examples/traefik/librenms.env [hunk 1]
```diff
@@ -14,3 +14,5 @@ LIBRENMS_SNMP_COMMUNITY=librenmsdocker
 
 LIBRENMS_WEATHERMAP=false
 LIBRENMS_WEATHERMAP_SCHEDULE=*/5 * * * *
+
+SNMP_TRAP_MIB_DIRS=/opt/librenms/mibs:/opt/librenms/mibs/cisco
```
Comment (crazy-max) [inline examples/traefik/librenms.env:18]: Please remove changes from `librenms.env`, this is just an example and majority of users will prefer the default value.

**Review by crazy-max — CHANGES_REQUESTED** [review]
I don't think we should allow users to change the default value but adding extra dirs sgtm.

A var named `SNMP_EXTRA_MIB_DIRS` seems better. Value would be appended to the current one.

Also missing documentation in README.

File: examples/traefik/librenms.env (hunk 1 above)
Comment (hjcday) [reply examples/traefik/librenms.env]: Changes should reflect above. All example .env files removed reference. Readme updated. Env Var name updated to be more specific.

File: rootfs/etc/cont-init.d/08-svc-snmptrapd.sh [hunk 2]
```diff
@@ -42,6 +42,6 @@ mkdir -p /etc/services.d/snmptrapd
 cat >/etc/services.d/snmptrapd/run <<EOL
 #!/usr/bin/execlineb -P
 with-contenv
-/usr/sbin/snmptrapd -f -m ALL -M /opt/librenms/mibs:/opt/librenms/mibs/cisco udp:162 tcp:162
+/usr/sbin/snmptrapd -f -m ALL -M /opt/librenms/mibs:/opt/librenms/mibs/cisco${SNMP_EXTRA_MIB_DIRS} udp:162 tcp:162
```
Comment (murrant) [inline rootfs/etc/cont-init.d/08-svc-snmptrapd.sh:45]: Should this prepend `:` if the variable is not empty?

Comment (hjcday) [reply rootfs/etc/cont-init.d/08-svc-snmptrapd.sh:45]: Not sure if this was going to cause an issue or not. I put in the readme example to include the : before the dir, if this is sufficient

Comment (hjcday) [reply rootfs/etc/cont-init.d/08-svc-snmptrapd.sh]: Put the ':' back in and tested with both variable with string and empty - both worked. Should be good to go

**Comment by hjcday** [comment]
@crazy-max should be good to go now

**Review by crazy-max — CHANGES_REQUESTED** [review]
Please quash your commits, otherwise LGTM.

**Comment by hjcday** [comment]
> Please squash your commits, otherwise LGTM.

I hope this is done correctly..

**Comment by crazy-max** [comment]
> > Please squash your commits, otherwise LGTM.
> 
> I hope this is done correctly..

There is still unrelated changes like https://github.com/librenms/docker/pull/354/commits/21836147075cfb3074dea607a9caa681dbf6965b. The merge commit looks to be the issue, there should be a single commit when squashed: https://github.com/librenms/docker/pull/354/commits

**Pull request closed without merging**